src/llmq/context.cpp

```diff
@@ -19,45 +19,53 @@
 #include <llmq/utils.h>
 #include <masternode/sync.h>
 
-LLMQContext::LLMQContext(CEvoDB& evo_db, CTxMemPool& mempool, CConnman& connman, CSporkManager& sporkman,
-                         const std::unique_ptr<PeerManager>& peerman, bool unit_tests, bool wipe) {
-    Create(evo_db, mempool, connman, sporkman, peerman, unit_tests, wipe);
-
-    /* Context aliases to globals used by the LLMQ system */
-    quorum_block_processor = llmq::quorumBlockProcessor.get();
-    qman = llmq::quorumManager.get();
-    clhandler = llmq::chainLocksHandler.get();
-    isman = llmq::quorumInstantSendManager.get();
+namespace {
+llmq::CQuorumBlockProcessor* InitQuorumBlockProcessor(CEvoDB& evo_db, CConnman& connman, const std::unique_ptr<PeerManager>& peerman)
```

Comment:
Is `InitQuorumBlockProcessor` going to re-used anywhere else? What are the benefits of it being a function?

Reply:
to make RAII initialization inside constructor, to make init std::unique and internal pointer all together.

Reply:
Is there anything preventing us from using this approach uniformly across all applicable members?